### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by bornagaintechnology** [gottcode/focuswriter#157]
I am new to GitHub so I'm not sure if this is where I post this but I noticed there is no help file or documentation. This is something I'd be glad to help with. I've never done documentation before but I would love to learn.

Josh Jensen
Owner/Operator at Born Again Technology

**Comment by gottcode** [comment]
What kind of documentation did you have in mind? A user manual? A wiki? I am so involved with the program that I have a hard time trying to describe it from a user's point of view, and I would appreciate help. I would need to see a sample of what you would want to say before signing off on it, but that sounds great! Thanks!

**Comment by bornagaintechnology** [comment]
I would love to help and give back because I'm really liking the software. I was thinking of working on a user manual but I like your idea of a wiki which you could provide a link in your help menu. Documentation is not something I've ever done but I'd like to put my writing skills to use. I think a wiki would be the most democratic, open-source way so if others wanted to contribute they could. I looked up some examples and Blender has a really good one. http://wiki.blender.org/ 

What I need from you is to install MediaWiki on a subdomain of your site and give me administrator privileges. A good link would probably be wiki.gottcode.org That way if you choose to have wiki's for your other software programs you could. 

As for your signing off on anything I write up, maybe there is functionality for that in MediaWiki. I'm not sure, I'm new to that software.

Additionally, I'm wondering if you have any plans to help writers organize research. When writing non-fiction and fiction alike having easy access and a way to organize research is paramount. Have you ever checked out the program Scrivener? I like it but it's not open source and I like to shy away from proprietary software.

I look forward to working with you!

**Comment by gottcode** [comment]
> I would love to help and give back because I'm really liking the software. I was thinking of working on a user manual but I like your idea of a wiki which you could provide a link in your help menu. Documentation is not something I've ever done but I'd like to put my writing skills to use. I think a wiki would be the most democratic, open-source way so if others wanted to contribute they could. I looked up some examples and Blender has a really good one. http://wiki.blender.org/

> What I need from you is to install MediaWiki on a subdomain of your site and give me administrator privileges. A good link would probably be wiki.gottcode.org That way if you choose to have wiki's for your other software programs you could.

Actually, I much prefer it when programs have user manuals, not wikis. I was only trying to figure out what you were asking for. If I was to use a wiki, which I am not sold on, it would not be free-for-all, and it would not be hosted on gottcode.org. I would simply use the GitHub Wiki support, and add users I feel are qualified for writing the documentation. Again, though, my programs are small, and a simple user manual would probably suffice.

> As for your signing off on anything I write up, maybe there is functionality for that in MediaWiki. I'm not sure, I'm new to that software.

I think you misunderstood, I just want to see a sample of what you are planning on writing before I add your documentation.

> Additionally, I'm wondering if you have any plans to help writers organize research. When writing non-fiction and fiction alike having easy access and a way to organize research is paramount. Have you ever checked out the program Scrivener? I like it but it's not open source and I like to shy away from proprietary software.

In the future, please open separate feature requests. I have not used Scrivener, but what you are describing is quite complicated and far outside the scope of FocusWriter, sorry. I will most likely not be adding anything like that.

**Comment by bornagaintechnology** [comment]
Yes, I misunderstood. I thought you were suggesting a wiki. I would be more than happy to write up a user manual. I'll get a start on it and e-mail you what I have so you can provide feedback.
